end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<p>The plot is too crowded and the legend is blocking the plot.</p><p>Also you need to add explanation of what this plot is presenting.</p><p>Put labels on the y-axis and the x-axis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Agreed.

Reply:
<p>I think I sent the wrong version. I will resend/pull.</p>